napari/utils/info.py

```diff
@@ -248,11 +285,14 @@ def sys_info(as_html: bool = False) -> str:
     text += f'  - Triangulation backend: {_triangulation_backend}<br>'
 
     text += '<br><b>Settings path:</b><br>'
-    text += f'  - {_config_path}'
+    text += f'  - {_config_path}<br>'
 
     text += '<br><b>Launch command</b><br>'
```

Comment:
```suggestion
    text += '<br><b>Launch command:</b><br>'
```
As noticed by Tim, to match the others